report.js

```diff
@@ -1155,36 +1169,37 @@ function process_burnchain_ops() {
         }
       }
     }
-    console.log("Statistics ========================================================================================================================")
-    console.log("miners (last block):", miner_count_last_block)
-    console.log("miners (overall):", Object.keys(miners).length)
-    console.log("total commit (last block):", numberWithCommas(burn_last_block, 0), "sats")
-    console.log("block reward (last block):", numberWithCommas(reward_last_block, 2), "STX")
-    console.log("btc blocks:", blocks)
-    console.log("empty btc blocks:", empty_blocks)
-    console.log("actual_win_total:", actual_win_total)
-    console.log("orphaned blocks:", blocks - empty_blocks - actual_win_total - incorrect_blocks)
-    console.log("incorrect blocks:", incorrect_blocks)
-    if (use_txs) {
-      console.log("total transactions (excl coinbase)", transaction_count)
-    }
-    if (show_distances) {
-      console.log("block_parent_distances")
-      for (let index = 0; index < block_parent_distances.length; index++) {
-        const block_parent_distance = block_parent_distances[index]
-        if (block_parent_distance) {
-          console.log(`${index} ${block_parent_distance} ${(block_parent_distance / block_parent_distance_count * 100).toFixed(2)}%`)
-        }
+    if (show_totals) {
+      console.log("Statistics ========================================================================================================================")
+      console.log("miners (last block):", miner_count_last_block)
+      console.log("miners (overall):", Object.keys(miners).length)
+      console.log("total commit (last block):", numberWithCommas(burn_last_block, 0), "sats")
+      console.log("block reward (last block):", numberWithCommas(reward_last_block, 2), "STX")
+      console.log("btc blocks:", blocks)
+      console.log("empty btc blocks:", empty_blocks)
+      console.log("actual_win_total:", actual_win_total)
+      console.log("orphaned blocks:", blocks - empty_blocks - actual_win_total - incorrect_blocks)
+      console.log("incorrect blocks:", incorrect_blocks)
+      if (use_txs) {
+        console.log("total transactions (excl coinbase)", transaction_count)
       }
-      console.log("block_commits_parent_distances")
-      for (let index = 0; index < block_commits_parent_distances.length; index++) {
-        const block_commits_parent_distance = block_commits_parent_distances[index]
-        if (block_commits_parent_distance) {
-          console.log(`${index} ${block_commits_parent_distance} ${(block_commits_parent_distance / block_commits_parent_distance_count * 100).toFixed(2)}%`)
+      if (show_distances) {
```

Comment:
why do you have `show_distances` as part of `show_totals`, this should be independent

Reply:
It was based on the original code, in your version you can see `show_distances` is separate on L1149:

https://github.com/psq/stacks-dump/blob/57a82caafaf4242a68f8757b91c97121c3d09719/report.js#L1149-L1157

Then referenced again as part of show totals:

https://github.com/psq/stacks-dump/blob/57a82caafaf4242a68f8757b91c97121c3d09719/report.js#L1171-L1186

In testing the `-d, --distances` flag I can see this is not the desired output. I thought it was part of the total stats.

Moved the separate code up to `show_distances` to fix in 176a14d.